BIC_codes/functions/dFC_funcs.py

```diff
@@ -0,0 +1,3472 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Jun 13 22:34:49 2021
+
+@author: mte
+"""
+
```

Comment:
you could add a setup.py to make it possible to install the package so you don't have to manipulate the python path in your notebook